draft-ietf-quic-manageability.md

```diff
@@ -493,7 +493,7 @@ to on-path observers.
 Version Negotiation packets are used by the server to indicate that a requested
 version from the client is not supported (see {{Section 6 of QUIC-TRANSPORT}}.
 Version Negotiation packets are not intrinsically protected, but future QUIC
-versions will use later encrypted messages to verify that they were authentic.
+versions may use later encrypted messages to verify that they were authentic.
```

Comment:
```suggestion
versions could use later encrypted messages to verify that they were authentic.
```

We have evidence that they will though: draft-ietf-quic-version-negotiation, so I'm not sure that this is something that we need to prevaricate on.

Reply:
that is even better, we can tell it more substantiate way..